### PR TITLE
Add question for approach aid type

### PR DESCRIPTION
**Pull request by 7h30n3**

### CODE REVIEW

File: assets/datasets/question_catalog.json [hunk 1]
```diff
@@ -846,6 +846,46 @@
...
+            "edge_recess": {
+               "name": "Aussparung",
+               "image": "assets/images/questions/1201a.jpg",
+               "osm_tags": {
+                  "kerb:approach_aid": "edge_recess"
+               }
+            },
+            "lane_guidance": {
+               "name": "Spurführung",
+               "image": "assets/images/questions/1201b.jpg",
```
Comment (Robbendebiene) [inline assets/datasets/question_catalog.json:866]: Couldn't we already use the existing `assets/images/questions/1200a.jpg` here`

File: assets/datasets/question_catalog.json [hunk 2]
```diff
@@ -846,6 +846,46 @@
          }
       ]
    },
+   {
+      "id": 1201,
+      "name": "Anfahrtshilfen-Art",
+      "question": "Wie sieht die Anfahrtshilfe für Busse aus?",
+      "description": "Bordsteine mit Spurführung sind speziell geformt, um Bussen ein dichtes Heranfahren an die Bordsteinkante zu ermöglichen.",
```
Comment (Robbendebiene) [inline assets/datasets/question_catalog.json:853]: "Bordsteine mit Spurführung..." -> "Bordsteine mit Anfahrtshilfe..." for consitency

Comment (7h30n3) [reply assets/datasets/question_catalog.json]: Fixed.

**Comment by 7h30n3** [comment]
Now only appropriate photos are missing. But I think we can add them later.

**Pull request merged**